config/config.go

```diff
@@ -17,12 +17,13 @@ type Configuration struct {
 
 // Config contains the information required by junos_collector to create SSH based NETCONF connections.
 type Config struct {
-	Username       string   `yaml:"username"`
-	Timeout        int      `yaml:"timeout"`
-	Password       string   `yaml:"password"`
-	SSHKey         string   `yaml:"ssh_key"`
-	AllowedTargets []string `yaml:"allowed_targets"`
-	Collectors     []string `yaml:"enabled_collectors"`
+	Username          string   `yaml:"username"`
+	Timeout           int      `yaml:"timeout"`
+	Password          string   `yaml:"password"`
+	SSHKey            string   `yaml:"ssh_key"`
+	AllowedTargets    []string `yaml:"allowed_targets"`
+	Collectors        []string `yaml:"enabled_collectors"`
+	InterfaceDescKeys []string `yaml:"interface_description_keys"`
```

Comment:
i tacked this onto this struct, let me know if you'd rather have somewhere else or broken out